src/main/java/com/blogify/blogapi/model/exception/ApiException.java

```diff
@@ -0,0 +1,15 @@
+package com.blogify.blogapi.model.exception;
+
+public class ApiException extends RuntimeException {
+  private final ExceptionType type;
```

Comment:
when u make it final, can u use, this.type = type?

Reply:
You can assign it _INITIALIZE_ it _ONCE_ in the constructor
so, yeah you can this.type = type in the constructor